examples/api/lib/widgets/overlay/overlay_portal.0.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for OverlayPortal
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: _title,
+      home: Scaffold(
+        appBar: AppBar(title: const Text('OverlayPortal Example')),
+        body: const Center(child: HoverWidget()),
+      )
+    );
+  }
+}
+
+class HoverWidget extends StatefulWidget {
+  const HoverWidget({super.key});
+
+  @override
+  State<StatefulWidget> createState() => HoverState();
+}
+
+class HoverState extends State<HoverWidget> {
+  bool isHovering = false;
+
+  void _onEnter(PointerEnterEvent event) {
+    setState(() { isHovering = true; });
+  }
+
+  void _onExit(PointerExitEvent event) {
+    setState(() { isHovering = false; });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTextStyle(
+      style: DefaultTextStyle.of(context).style.copyWith(fontSize: 50),
+      child: MouseRegion(
+        onEnter: _onEnter,
```

Comment:
Will "mouse" region behave well for mobile phones? If not, maybe we can improve this example since flutter targets on mobiles as well?

Reply:
This is an interactive example in the documentation and will be shown in a browser so I think in most cases developers will have a mouse connected while checking out the example.

Reply:
But yeah you're right I'll change it to a button so it's more accessible.